Automattic/jetpack
# Module Descriptions: add 3 missing modules.

Fixes #11711

#### Changes proposed in this Pull Request:

* add 3 missing modules to the module info list.
* fix all phpcs warnings.
* add file to list of autolint files.

#### Testing instructions:

* Go to Jetpack > Debug in your dashboard.
* Click the link to view all modules at the bottom of the page.
* Make sure Comment Likes, PWA, and Search all link to their respective support docs.

#### Proposed changelog entry for your changes:

* None
